warp10/src/main/java/io/warp10/continuum/Configuration.java

```diff
@@ -440,7 +440,12 @@
    * Port on which the DirectoryService will listen
    */
   public static final String DIRECTORY_PORT = "directory.port";
-  
+
+  /**
+   * Port on which the DirectoryService will listen
```

Comment:
Please correct the javadoc comment.

Also document the newly added configuration keys into the configuration templates for both distributed and standalone.